waku/v2/node/config.nim

```diff
@@ -129,6 +129,11 @@ type
       defaultValue: ""
       name: "filternode" }: string
     
+    filterTimeout* {.
+      desc: "Timeout for filter node in seconds.",
+      defaultValue: 60 * 60 * 24
+      name: "filter-timeout" }: float
```

Comment:
Any specific reason this is a float and not e.g. uint16? I don't think we need to specify subseconds :P

Reply:
epochTime() or getTime() in nim is represented in float, hence the config is in float as well.

Reply:
Couldn't this be a `Duration`? (https://github.com/status-im/nim-chronos/blob/master/chronos/timer.nim#L185)

Reply:
solved in 2f0e923b3e6c2494bdd639ee6ad352f56bd2f434